lxd/vm_qemu.go

```diff
@@ -2008,6 +2009,7 @@ func (vm *vmQemu) FilePush(fileType string, srcPath string, dstPath string, uid
 	if err != nil {
 		return err
 	}
+	agent.Disconnect()
```

Comment:
Shouldn't that be a defer?

Reply:
@stgraber indeed it should, although this didnt show up in my testing as FilePush doesnt actually wait on the channel that Disconnect() closes so it works fine, but I added what should have been defer agent.Disconnect() to all uses of agent for completeness. Fixed.